src/ringcentral-web-phone.js

```diff
@@ -237,6 +251,10 @@
 
         this.userAgent.media = {};
 
+        this.userAgent.enableQos = options.enableQos || false;
```

Comment:
Since you do `session.on('SessionDescriptionHandler-created',function () {` and options object is available in this scope there's no need for `this.userAgent.enableQos = options.enableQos || false;`